### ISSUE TITLE
Allow registered users to bypass banned IP ranges (GNA #17462)

### ISSUE THREAD

**Issue by wesnoth-bugs** [wesnoth/wesnoth#1248]
**[Original submission](https://gna.org/bugs/index.php?17462)** by wintermute on 2011-01-11

Some legitimate users are being blocked by the ranged IP bans put in place to deal with some of the most troublesome individuals.  The multiplayer moderators suggest that we allow users to bypass the ban range if they have registered their account.  This would be a straightforward way to deal with users that have been blocked unintentionally.

(Reproduced on all)
Release: all
Priority: 5 - Normal
Severity: 3 - Normal

**Comment by wesnoth-bugs** [comment]
**Modified on 2011-01-18**

**soliton wrote:**

> See also bug #17491.
> 
> 
> 
> One point to add is that we then need to check whether the account is banned on the forum or separately marked as banned in mp. Since we don't want to let the real trouble maker get around the ban of course. ;-)

**Comment by ProditorMagnus** [comment]
This issue needs to be reconsidered now that everyone is registered.

**Comment by CelticMinstrel** [comment]
I don't think the main wesnoth server requiring everyone to be registered is sufficient to mark this as "won't fix". There can always still be third-party servers that don't have that requirement.

**Comment by jostephd** [comment]
@CelticMinstrel I think it's fine to close this issues now that the original use case has been addressed. If someone administers a private wesnoth server and needs this, they can come and ask to reopen this issue, just like they'd have filed an issue if they needed any other new feature implemented.
